.github/workflows/ci.yml

```diff
@@ -70,6 +70,43 @@ jobs:
     env:
       RUST_BACKTRACE: 1
 
+  check-msrv:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+      - name: Set up Python
+        uses: actions/setup-python@v2
+        with:
+          python-version: 3.8
+      - name: Install MSRV Rust
+        uses: actions-rs/toolchain@v1
+        with:
+          profile: minimal
+          toolchain: stable
```

Comment:
Why is this `stable` instead of `1.41.1`?

Reply:
Oops, I mistakenly changed this when I added nightly for `-Z minimal-versions`.
Thanks!